core/templates/pages/community-dashboard-page/question-opportunities/question-opportunities.directive.ts

```diff
@@ -234,9 +220,13 @@ angular.module('oppia').directive('questionOpportunities', [
                   };
                   // Checking if Question contains all requirements to enable
                   // Save and Publish Question
-                  $scope.isSaveButtonDisabled = function() {
-                    return $scope.question.validate(
-                      $scope.misconceptionsBySkill);
+                  $scope.isQuestionValid = function() {
```

Comment:
This logic is repeated [here](https://github.com/oppia/oppia/pull/9337/files/56e8997223cd73978ca3c8de18ff2df908f27d3a..2ae6082d5b36e57a2cc732339b303fda4ca40e5b#diff-bf49205f6cc835acaeeedef2d12b08a3R682). Can we de-duplicate (perhaps by moving the functionality to a common service)?